meetings/2021-06-16.md

```diff
@@ -0,0 +1,151 @@
+# 2021-06-16 Solid Authorization
+
+https://meet.jit.si/solid-authorization
+
+
+## Agenda
+
+* Minutes https://github.com/solid/authorization-panel/pull/221
+    * Previous week actions
+* Follow up on previous week's UMA discussion
+* [UC3 Inheritance](https://github.com/solid/authorization-panel/)
+* [JWK in Access Control](https://github.com/solid/authorization-panel/discussions/225)
+* [Proofs and Origins](https://github.com/solid/authorization-panel/discussions/223)
+
+## Present
+
+* Aaron Coburn
+* Henry Story
+* Justin B
+* Elf P
+* Matthieu B
+* Eric P
+* Martynas Jusevicius
+
+
+## Minutes
+
+### UMA discussion
+
+Justin: [last week](2021-06-09.md) we went through slides for consent-based workflows. Given an agent/actor who want to give access to another app/user. A trusted authorization agent helps execute these workflows. This is a general pattern, but the interaction/protocol hasn't been specified.
+
+Justin: the interop spec just states: "apply permissions". A working, deployed private implementation with custom code (not using UMA, just secure redirects).
+
+Justin: UMA is an implementation of this pattern with good adoption. Designed to work with the OAuth2 stack and complements some of the other standards we are using in Solid.
+
+Justin: this is something we can look at as a trusted pattern. Not saying "UMA or bust", but thinks we should look at the problems it can help us with.
```

Comment:
```suggestion
Justin: this is something we can look at as a trusted pattern. Not saying "UMA or bust", but think we should look at the problems it can help us with.
```